src/ImageItem.js

```diff
@@ -0,0 +1,19 @@
+import React, { Component } from 'react'
+
+export default class ImageItem extends Component {
+    render() {
+        return <li>
+            <div className="info-container">
```

Comment:
"I really wanted to contain each of the "creatures" and "horned creatures"... However, whenever I added a width, it broke everything else so I eventually gave up"
I'm not sure where you were adding the width, but if you add it to the info-container div, it contains the text nicely.
.info-container {
    width: 300px;
}